src/libs/actions/Card.js

```diff
@@ -0,0 +1,49 @@
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '../../ONYXKEYS';
+import * as API from '../API';
+
+/**
+ * @param {Number} cardID
+ */
+function reportDigitalExpensifyCardFraud(cardID) {
```

Comment:
If I am not mistaken, we [decided to rebrand these as "virtual cards"](https://github.com/Expensify/Expensify/issues/280361#issuecomment-1653357660). Could we update that for consistency? So...

- `reportDigitalExpensifyCardFraud` -> `reportVirtualExpensifyCardFraud`
- `ReportDigitalExpensifyCardFraud` -> `ReportVirtualExpensifyCardFraud`
- `report-digital-fraud` -> `report-virtual-fraud`
- etc

Reply:
Done!